src/audio/mixin_mixout/mixin_mixout.c

```diff
@@ -123,6 +123,7 @@ static int mixin_init(struct processing_module *mod)
 	dev->ipc_config.frame_fmt = valid_fmt;
 	mod->skip_src_buffer_invalidate = true;
 
+	mod->max_sinks = MIXIN_MAX_SINKS;
```

Comment:
Would this be a Kconfig or is it set in the IPC4 ABI ?

Reply:
@lgirdwood it is set in the mixin/mixout header and doesn't seem to be bound by ABI. But the mixin_mixout module only works with IPC4 so I guess indirectly it is